src/modules/api_implementation.js

```diff
@@ -0,0 +1,34 @@
+export default class KrakenAPI {
+  static baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+
+  static getScores = async (gameID) => fetch(`${this.baseURL}/games/${gameID}/scores/`, {
+    method: 'GET',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then(async (response) => {
+    if (response.status === 200) {
+      const data = await response.json();
+      return data.result;
+    }
+    return null;
+  });
```

Comment:
[OPTIONAL]

- You have done great in this project 👍🏽 . I just wanted to point out here that since you are using async/await, there is no need to use chaining as you have done by using `.then` as this was removed in async/await as an improvement to callbacks and Promises.

I think you should [read](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) more on how to use async/await for API calls.

Reply:
Thanks for the review and the approval  I will kindly check the correct syntax of  async and await to improve my code 👏@vickymarz